core/tokens.js

```diff
@@ -0,0 +1,152 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const StyleDictionary = require('style-dictionary');
```

Comment:
Can we add a sentence or two about why we are doing this? For example, it would be good to document why we use `style-dictionary` as opposed to just hardcoding these values directly.

Reply:
Done!